packages/connect/package.json

```diff
@@ -42,14 +42,15 @@
     "smoldot": "0.2.9"
   },
   "devDependencies": {
-    "@substrate/smoldot-test-utils": "file:../smoldot-test-utils",
+    "@substrate/smoldot-test-utils": "^0.1.0",
     "@types/jest": "^26.0.20",
     "@types/jsdom": "^16.2.7",
     "@types/mkdirp": "^1.0.1",
     "@typescript-eslint/parser": "^4.27.0",
     "eslint": "^7.21.0",
     "eslint-plugin-tsdoc": "^0.2.14",
     "jest": "^27.0.0-next.9",
+    "jest-silent-reporter": "^0.5.0",
```

Comment:
why is this added? I dont thing we need this for any reason

Reply:
Its the reporter used to minimse the test outputs so that it passes silently but fails on failing tests.